airflow/serialization/serialized_objects.py

```diff
@@ -64,6 +63,7 @@
     BaseAsset,
 )
 from airflow.sdk.definitions.baseoperator import BaseOperator as TaskSDKBaseOperator
+from airflow.sdk.definitions.baseoperatorlink import XComOperatorLink
 from airflow.sdk.definitions.mappedoperator import MappedOperator
 from airflow.sdk.definitions.param import Param, ParamsDict
 from airflow.sdk.definitions.taskgroup import MappedTaskGroup, TaskGroup
```

Comment:
We should decide (not in this PR) how we want to import these inside core -- do we import from individual files, or should it all be from `airflow.sdk` only.

Things we expect users to need vs "internal" things needs some consideration.

Reply:
I will create a discussion / issue prior to merge

Reply:
Created a discussion for this: https://github.com/apache/airflow/discussions/47486, will await responses on it.